clarity_ext/domain/artifact.py

```diff
@@ -0,0 +1,49 @@
+from clarity_ext.domain import Analyte
+from clarity_ext.domain import Container
+
+
```

Comment:
I think you're right about that. It's valid to hide the fact that there's an input/output map behind this - user's don't really need to know that. It's also good to shorten the name.

I'll change it to `StepRepository` with the intent that all data available in a step can be fetched through it in the future. We could change it to `StepArtifactRepository` if the class starts being too large.